specifications/models/caffe2/mobilenet_v2/mobilenet_v2_quant_taskset_c.json

```diff
@@ -0,0 +1,33 @@
+{
+  "model": {
+    "category": "CNN",
+    "description": "Trained MobileNet v2 on Caffe2",
+    "files": {
+      "init": {
+        "filename": "init_net.pb",
+        "location": "https://s3.amazonaws.com/download.caffe2.ai/models/mobilenet_v2_quant/init_net.pb",
+        "md5": "c503fcfbac53d8a3b4a5b716e898b59e"
+      },
+      "predict": {
+        "filename": "predict_net.pb",
+        "location": "https://s3.amazonaws.com/download.caffe2.ai/models/mobilenet_v2_quant/predict_net.pb",
+        "md5": "dc2dba5ef9ed216678558301f4690063"
+      }
+    },
+    "format": "caffe2",
+    "kind": "deployment",
+    "name": "mobilenet_v2_quant_taskset_c"
+  },
+  "tests": [
+    {
+      "arguments": "{program} --net {files.predict} --init_net {files.init} --warmup {warmup} --iter {iter} --input \"data\" --input_dims \"1,3,224,224\" --input_type float --run_individual true",
```

Comment:
This will fail, won't it? When adding {program}, need to change the filed name to "command"